bims/templates/includes/captcha.html

```diff
@@ -0,0 +1,26 @@
+<div id="captcha-info"><span id="warning-captcha">Please verify that you are not a bot first.</span><br/></div>
+<div class="g-recaptcha" data-callback="enableBtn"></div>
+
+{% if site_key != '' %}
+    <script src="https://www.google.com/recaptcha/api.js?onload=CaptchaCallback&render=explicit&hl=en"></script>
+ {% else %}
+    <script>
+        $(document).ready(function () {
+            enableBtn();
+        })
+    </script>
+{% endif %}
+
+<script type="text/javascript">
+    var CaptchaCallback = function(){
+      $('.g-recaptcha').each(function(){
+        grecaptcha.render(this,{'sitekey' : '{{ site_key }}' } );
+      })
+    };
+
+    function enableBtn(){
+        $('.btn-login').show();
```

Comment:
This means that this template can't be reused for other form, maybe create a function here to search a button that has specific class name e.g `btn-need-recaptcha`.

Reply:
done